package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "exceljs",
-  "version": "0.4.12",
+  "version": "0.4.13",
```

Comment:
Bumping the package version is managed by the release scripts (npm version patch, etc). Please can you revert this line